app.css

```diff
@@ -103,6 +104,28 @@ ul {
   width: 50%
 }
 
+.pane img.index-image {
+  width: 100%;
+}
+
+.pane.index-pane .index-media {
+  /* Align sub-items to top */
+  align-items: start;
+  display: flex;
+}
```

Comment:
It looks weird that the `<audio>` element doesn't take up the entire width of the `.index-media`.

![Screenshot_2021-03-10 Rehydrate](https://user-images.githubusercontent.com/324298/110731594-9fea5200-81f0-11eb-9943-791c5cc627c2.png)

Could you try something like

```.index-media .episode-player {
  width: 100%;
  border-top-left-radius: 4px;
  border-top-right-radius: 4px;
}```

(This will also take care of two of the corners being rounded and two being unrounded.)

Reply:
done!